mssqlscripter/main.py

```diff
@@ -92,6 +92,7 @@ def main(args):
                 scriptercallbacks.handle_response(response, parameters.DisplayProgress)
 
         # Only write to stdout if user did not provide a file path.
+        logger.info('Using encoding: {}'.format(sys.stdout.encoding))
```

Comment:
Can you add 'stdout' to the message so it's clear in the logs what the encoding refers to?